### PR TITLE
profile_solvers improvements

### PR DESCRIPTION
**Pull request by abelsiqueira**
- Add width and height in `profile_solvers`;
- Remove excessive xlabel and ylabel in `profile_solvers`;
- Change `import Plots` to `using Plots` so `Plots.foo` changes to `foo`.

### CODE REVIEW

File: src/profiles.jl [hunk 1]
```diff
@@ -36,49 +36,76 @@ Inputs:
...
+  ncosts = length(costs)
+  npairs = div(nsolvers * (nsolvers - 1), 2)
 
   # profiles with all solvers
   ps = [performance_profile(Ps[1], string.(solvers), title=costnames[1], legend=:bottomright)]
-  for k = 2 : length(Ps)
-    push!(ps, performance_profile(Ps[k], string.(solvers), title=costnames[k], legend=false))
+  if nsolvers > 2
+    xlabel!(ps[1], "")
+  end
```
Comment (dpo) [inline src/profiles.jl]: Could we make this a one liner: `nsolver > 2 && xlabel!(ps[1], "")`?

File: src/profiles.jl [hunk 2]
```diff
@@ -36,49 +36,76 @@ Inputs:
...
+  for k = 2 : ncosts
+    p = performance_profile(Ps[k], string.(solvers), title=costnames[k], legend=false)
+    if nsolvers > 2
+      xlabel!(p, "")
+    end
```
Comment (dpo) [inline src/profiles.jl]: One liner here too?

File: src/profiles.jl [hunk 3]
```diff
@@ -36,49 +36,76 @@ Inputs:
...
         pair = [solvers[i], solvers[j]]
         dfs = (stats[solver] for solver in pair)
         Ps = [hcat([cost(df) for df in dfs]...) for cost in costs]
 
         clrs = [colors[i], colors[j]]
-        push!(ps, performance_profile(Ps[1], string.(pair), palette=clrs, legend=:bottomright))
+        p = performance_profile(Ps[1], string.(pair), palette=clrs, legend=:bottomright)
+        if ipairs < npairs
+          xlabel!(p, "")
+        end
```
Comment (dpo) [inline src/profiles.jl]: And here.

File: src/profiles.jl [hunk 4]
```diff
@@ -36,49 +36,76 @@ Inputs:
...
+        push!(ps, p)
         for k = 2 : length(Ps)
-          push!(ps, performance_profile(Ps[k], string.(pair), palette=clrs, legend=false))
+          p = performance_profile(Ps[k], string.(pair), palette=clrs, legend=false)
+          if ipairs < npairs
+            xlabel!(p, "")
+          end
```
Comment (dpo) [inline src/profiles.jl]: And here.

File: src/profiles.jl (hunk 1 above)
Comment (abelsiqueira) [reply src/profiles.jl]: Done

**Comment by dpo** [comment]
Thank you!

**Pull request merged**